src/modules/transactions/actions/add-transactions.js

```diff
@@ -85,7 +86,12 @@ function buildTradeTransactionGroup(group, marketsData) {
     header.transactions[0].maker
   ) {
     // own order filled
-    header.message = formatTransactionMessage(sumBuy, sumSell, "Order", true);
+    header.message = formatTransactioFillMessage(
+      header.transactions[0].type,
+      header.transactions[0].amount,
+      header.transactions[0].price,
+      header.transactions[0].meta && header.transactions[0].meta.outcome
+    );
   } else {
     header.message = formatTransactionMessage(sumBuy, sumSell, "Trade");
```

Comment:
I think we should be calling `formatTransactionFillMessage` here as well. The text should be the same regardless of whether the person filling the trade is filling their own order or someone else's.